pkg/service/auth_context.go

```diff
@@ -249,3 +249,25 @@ func (authContext *AuthContext) GetResolvedMetadata() map[interface{}]interface{
 	}
 	return m
 }
+
+type authContextData struct {
+	Request *envoy_auth.AttributeContext `json:"context"`
+	Context map[string]interface{}       `json:"auth"`
+}
```

Comment:
I'd expect some different structure holding "data", it's a bit puzzling constructing this with a `Request` and a `Context`, specially this last one not being of the type of `context`

Reply:
I hid this structure as implementation detail of `AuthContext.GetDataForAuthorization()` and it is now:

```go
Context  *envoy_auth.AttributeContext `json:"context"`
AuthData map[string]interface{}       `json:"auth"`
```

I hope it's clearer.